notebooks/tutorials/2_pipeline_lesson10_qe/notebook.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
Did this file have content in it before the change?

Reply:
looks like this is an old commit (the correct changes were merged on Nov 9th: https://github.com/quantopian/research_public/pull/291/files) so we don't want this commit. I'll probably cherry-pick my changes into a fresh branch to avoid this commit.